src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/updatecontactdetails/UpdateContactDetailsAboutToSubmitHandler.java

```diff
@@ -113,4 +122,26 @@ private void considerContestedMiniFormA(FinremCaseDetails finremCaseDetails,
             finremCaseDetails.getData().setMiniFormA(document);
         }
     }
+
+    /*
+     * Repeat mid-event validation to protect Users navigating with browser controls.
+     * @param finremCaseDetails case details
+     * @return list of errors
+     */
+    private List<String> getPostCodeErrors(FinremCaseDetails finremCaseDetails) {
```

Comment:
shall we make a single method which is shared by both mid event handler and about-to-submit handler? I think our future developers might miss that.

Reply:
`    /*
     * Distinct mid-event handlers validate postcodes for Consented and Contested cases.
     * This blends validation from each, to protect Users from skipping validation with browser controls.
     * @param finremCaseDetails case details
     * @return list of errors
     */`